prompt/prompt-executor/prompt-executor-clients/prompt-executor-openrouter-client/build.gradle.kts

```diff
@@ -15,6 +15,8 @@ kotlin {
                 api(project(":agents:agents-tools"))
                 api(project(":agents:agents-utils"))
                 api(project(":prompt:prompt-executor:prompt-executor-clients"))
+                implementation(project(":prompt:prompt-executor:prompt-executor-clients:prompt-executor-anthropic-client"))
```

Comment:
Why did you add dependencies on Anthropic and Google clients? It should be independent from these clients